akka-typed-tests/src/test/scala/akka/typed/cluster/ClusterShardingApiSpec.scala

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.cluster
+
+import akka.cluster.sharding.ClusterShardingSettings
+import akka.typed.scaladsl.Actor
+import akka.typed.scaladsl.adapter._
+import akka.typed.{ ActorSystem }
+
+class ClusterShardingApiSpec {
```

Comment:
I'll work on this then, thanks @johanandren for stealing some of the work to be done here :)

Reply:
Remove the sharding stuff from this PR, since Konrad is working on that.